kafka/README.md

```diff
@@ -6,16 +6,36 @@ This document defines how to describe Kafka-specific information on AsyncAPI.
 
 ## Version
 
-Current version is `0.1.0`.
+Current version is `0.2.0`.
 
 
 <a name="server"></a>
 
 ## Server Binding Object
 
-This object MUST NOT contain any properties. Its name is reserved for future use.
+This object contains information about the server representation in Kafka.
+
+##### Fixed Fields
 
+Field Name | Type | Description | Applicability [default] | Constraints
```

Comment:
If the whole "Constraints" column is empty, shouldn't we just get rid of it entirely?

Reply:
I was doing that to avoid the merge conflict if both this and https://github.com/asyncapi/bindings/pull/55 get approved

Reply:
Oh, I see. Thanks for the explanation 👍